examples/hello-world/step-by-step/cifar10/data/download.py

```diff
@@ -53,6 +53,7 @@ def define_parser():
 
 def main(args):
     datasets.CIFAR10(root=args.dataset_path, train=True, download=True)
+    datasets.CIFAR10(root=args.dataset_path, train=False, download=True)
```

Comment:
Do we need duplicate lines?
I thought the first line is going to download the dataset?